run-create/action.yaml

```diff
@@ -0,0 +1,45 @@
+# action.yml
+name: "Create a test run in the Qase TMS"
+description: "Create a test run in the Qase TMS."
+inputs:
+  token:
+    description: "Token for Qase API"
+    required: true
+  project:
+    description: "Project code in Qase"
+    required: true
+  title:
+    description: "Title of the test run"
+    required: true
+  description:
+    description: "Description of the test run"
+    required: false
+  environment:
+    description: "Environment of the test run"
+    required: false
+  milestone:
+    description: "Milestone of the test run"
+    required: false
+  plan:
+    description: "Plan of the test run"
+    required: false
```

Comment:
Inputs can have types declared: sting, int, bool. If the type is wrong, GH will show an error when a workflow is pushed, which is better and earlier than if CLI throws an error trying to get an environment by ID.

Reply:
I don't see any information about types: https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#inputs

Reply:
Fixed